README.md

```diff
@@ -64,6 +64,21 @@ cp completion/kubie.fish ~/.config/fish/completions/
 
 Then reopen fish or source the file.
 
+
+#### Zsh
+
+Autocomplete with Zsh can either be installed by sourcing the file eg.:
+
+```bash
+source ./completion/kubie.zsh
+```
+
+Or by adding it to your fpath and renaming the completion file to _kubie eg.:
+```bash
+cp .completion/kubie.zsh ~/.zsh/completions
```

Comment:
```suggestion
cp ./completion/kubie.zsh ~/.zsh/_kubie
```

Reply:
@mhkarimi1383 the autocompletions are working for me just fine (also for `kubie ns` and for `kubie` itself). I've added some suggestions above to clarify how to make it work. More specific steps can be given if needed.